enhancements/monitoring/alerts-ui-management.md

```diff
@@ -0,0 +1,642 @@
+- The main difference is the scope of data presented, providing a centralized, aggregated view for a fleet of clusters compared to a single cluster's data.
+- Key features for a multi-cluster view include a **Centralized View**, a **Cluster-Specific Context** filter, and enhanced **RBAC** for managing alerts across multiple clusters.
+
+---
+
+### Feature Prioritization
+
+The features are prioritized using tags: **Must-Have**, **Should-Have**, **Could-Have**, and **Won't-Have**.
+
+**Must-Have Features:**
+- Tabs changes (Alerts, Management: Alert rules, Silence rules)
+- Create user-defined alert rules (Alert rules definition and Metadata)
+- Advanced filtering capabilities
+- Bulk actions: disable, edit labels, edit component
+- Duplicate and Delete alert rules
+- Incident tab
+
+**Should-Have Features:**
+- Add components
+- Saved filters
+- Alert and alert rule side drawer
+- Add "Resource" column for node alerts
+
+**Could-Have Features:**
+- Notifications (alertmanager receivers)
+- PromQL expression autocompletion and graph
+- "Save as draft" wizard
+- Alert rule history
+- Acknowledge alert
+- Filter by triggered date/time
+- Column management
+- Additional alert action items (View logs, Troubleshoot, etc.)
+- Generate a summary report
+- Generate a dashboard
+- Manage groups
+
+**Won't-Have Features:**
+- Alertmanager sub-tab
+
+### API Endpoints
+
+Base path: `/api/v1/alerting`
+
+Rule identity
+
+- A rule is uniquely identified by the tuple: `(namespace, prometheusrule, ruleName, specHash)`.
+- Canonical opaque identifier used below as `ruleId`:
+  - `/namespaces/{namespace}/prometheusrules/{prometheusrule}/rules/{ruleName}/specHash/{specHash}`.
+    - `specHash` is server‑generated from the rule spec to ensure uniqueness and stability. It is computed as SHA‑256 (hex, lowercase) of the normalized fields:
+      - `expr`: trimmed with consecutive whitespace collapsed
+      - `for`: normalized to seconds
+      - `labels`: Validate labels `key` and `value` meet Prometheus syntax and character rules; drop empty values; sort by key ascending; join as `key=value` lines separated by `\n`
+      - Concatenate the three parts with `\n---\n` separators, then hash the UTF‑8 bytes
+    - Clients must treat `specHash` as opaque (do not construct it). When `expr`/`for`/labels change, the server recomputes `specHash` and returns an updated `ruleId`.
+
+**Alerts (instances)**
+
+| Method | Path                      | Description                                                                                         | Notes |
+|--------|---------------------------|-----------------------------------------------------------------------------------------------------|-------|
+| GET    | `/alerts`                 | List alert instances (Pending / Firing / Silenced) with post‑relabel labels. Supports the filter parameters listed below. | **Required** — merges `AlertRelabelConfig` with platform alerts. |
```

Comment:
@simonpasquier @jgbernalp I would like to drop this API in favor of an metric that will report this information.
The metric would report all pending, firing and pending alerts after relabeling. They will include the rule_id and the value will report the activeAt its the value(timestamp_seconds).
I believe this would be faster and more scaleable if we consider the multi cluster use case.

Example of the metric output:
alerts_effective_active_at_timestamp_seconds{alertname=.., alertstatus=.., namespace-.., node=.., group=.., component=.., rule_id=..}
Would it be possible to collect this metric in shorter intervals to MCOA Thanos?
Please let me know if you agree.